readme.md

```diff
@@ -397,9 +398,9 @@ Default: `false`
 ###### request
 
 Type: `Function`<br>
-Default: `http.request` `https.request` *(Depending on the protocol)*
+Default: `http.request` `https.request` `http2wrapper.request` *(Depending on the protocol)*
 
-Custom request function. The main purpose of this is to [support HTTP2 using a wrapper](#experimental-http2-support).
+Custom request function. The main purpose of this is to [support HTTP2 using a wrapper](#http2-support).
```

Comment:
> The main purpose of this is to [support HTTP2 using a wrapper](#http2-support).

This is misleading as it's no built-in. I would just drop that sentence or document a different use-case.